modules/postman-templating/src/template-client.ts

```diff
@@ -37,13 +27,13 @@ export default class TemplateClient {
    * @param templateBody - template body
    * @param params - dict of param variables used for interpolation
    */
-  parseTemplate(
+  parseTemplate = (
```

Comment:
why change to arrow function?

Reply:
Sorry probably a mistake. Will change it back.